build/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM alpine:3.3
-RUN apk --update --no-cache add \
-	ca-certificates
+
+RUN apk add --no-cache ca-certificates && \
+	rm -rf /var/cache/apk/*
```

Comment:
I thought `--no-cache` which is new in alpine 3.3 does the same thing than that, no?